auth/api.py

```diff
@@ -0,0 +1,12 @@
+from flask.views import MethodView
+from flask import request, abort
+
+
+class AuthAPI(MethodView):
+    def __init__(self):
+        if request.method != 'POST':
+            abort(400)
```

Comment:
use 405 method not allowed

Reply:
It was my mistake. I fixed it on https://github.com/amolabs/amo-storage/pull/1/commits/5bb0d4eb8d5ff08f66c9f83a49c677ee0017defc